src/js/components/Meter/__tests__/__snapshots__/Meter-test.js.snap

```diff
@@ -60,7 +60,7 @@ exports[`Meter background 1`] = `
       fill="none"
       stroke="#DDDDDD"
       strokeLinecap="square"
-      strokeOpacity="0.4"
+      strokeOpacity={0.4}
```

Comment:
This doesn't look right. It looks like we're leaking JSX syntax into the DOM.

Reply:
correct. good catch. let me look into it.